src/components/Slider/SwiperButtons.jsx

```diff
@@ -0,0 +1,34 @@
+import { React } from "react";
+
+import { useSwiper } from "swiper/react";
+
+const SwiperButtons = () => {
+  const swiper = useSwiper();
+
+  return (
+    <div>
+      <div>
```

Comment:
Double div. Use fragment if you need to <></>

Reply:
completed